src/ocean/meta/types/Typedef.d

```diff
@@ -162,3 +162,25 @@ unittest
     mixin(Typedef!(int, "MyInt"));
     static assert (is(TypedefBaseType!(MyInt) == int));
 }
+
+/*******************************************************************************
+
+    Helper function to reduce typedef to its base value
+
+    Params:
+        T = type to reduce to base, must be generated by `Typedef`
+
+*******************************************************************************/
+
+TypedefBaseType!(T) typedefBase ( T ) ( T value )
```

Comment:
`castTypedefToBase` would be clearer (but a bit longer).

Reply:
`castToBase`?

Reply:
Given that it's in a module named `Typedef`, yep :+1: